installation/routines/setup_autohotspot.sh

```diff
@@ -1,119 +1,91 @@
-#!/usr/bin/env bash
```

Comment:
Don't we need this?

Reply:
True, we could think about removing them on all scripts in the "routines" folder.
But that should be done with the reordering @pabera had started.

I would just keep this to have less irrelevant changes

Reply:
Oh i got your questions wrong 🤦 
I thought you suggest to remove it. Id idn't see it was removed.

I will add it again, but the above remains valid ^^